src/rp2_common/pico_platform/include/pico/platform.h

```diff
@@ -153,14 +153,14 @@ extern "C" {
  *
  * For example a `uint32_t` foo that will retain its value if the program is restarted by reset.
  *
- *     uint32_t __uninitialized_ram("my_group_name") foo;
+ *     uint32_t __uninitialized_ram("foo");
  *
  * The section attribute is `.uninitialized_ram.<group>`
```

Comment:
Does this comment need updating to `.uninitialized_data.<group>` to match the change a few lines below, or am I misunderstanding? :shrug:   (I don't know anything about linker stuff)

Reply:
no that comment was correct, the one i changed wasn;t